.github/workflows/pull-request.yml

```diff
@@ -21,3 +21,13 @@ jobs:
         uses: ./.github/actions/set-up-environment
       - name: Run tests
         run: npm test
+  playwright_tests:
+    needs: unit_tests
```

Comment:
Should we run the Playwright tests in parallel with the unit tests?

Reply:
```suggestion
    needs: build
```

Reply:
Yeah, happy to go with that. My initial thought was they follow on from each other but there's nothing worse than having to run tests multiple times just to find all of the failures.